src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -43,7 +43,7 @@ try
         "dev15.0.x" { } 
         "master" { } 
         "post-dev15" { } 
-        "features/refout" { }
+        "features_refout" { }
```

Comment:
I wasn't aware of that limitation.  Prehaps add a comment here to warn future developers that `/` will be converted to `_`